src/main/resources/data/survivalgames/games/nether.json

```diff
@@ -0,0 +1,42 @@
+{
+  "type": "survivalgames:survivalgames",
+  "name": "Survival Games (Nether)",
+  "dimension": "minecraft:the_nether",
+  "border": {
+    "start_size": 512,
+    "end_size": 16,
+    "safe_secs": 120,
+    "shrink_secs": 240
+  },
+  "players": {
+    "min": 1,
+    "max": 100
+  },
+  "town_depth": -6,
+  "outskirts_building_count": 0,
+  "nether_building_count": 10,
```

Comment:
Should be generified into "outskirts_building_count" and "outskirts_pool" as stated above